Index: target/classes/plugin.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>name: FarmInventory\r\nmain: cc.ranmc.farm.inventory.Main\r\nwebsite: https://www.ranmc.cc/\r\ndescription: '阿然定制24k纯钛合金超级无敌终终终终极版桃花源专属作物菜单插件'\r\nauthor: Ranica\r\nversion: Beta 2.1\r\napi-version: \"1.20\"\r\nsoftdepend: [PlaceholderAPI]\r\nfolia-supported: true\r\n\r\npermissions: \r\n  fm.user:\r\n    default: true\r\n    description: 'user only'\r\n\r\n  fm.admin:\r\n    default: op\r\n    description: 'admin only'\r\n\r\ncommands:\r\n  fm:\r\n    description: fm\r\n    usage: /<command>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/target/classes/plugin.yml b/target/classes/plugin.yml
--- a/target/classes/plugin.yml	(revision 13848cdc4b96488f521837f556359dc374fb6225)
+++ b/target/classes/plugin.yml	(date 1704059782086)
@@ -1,7 +1,7 @@
 name: FarmInventory
 main: cc.ranmc.farm.inventory.Main
 website: https://www.ranmc.cc/
-description: '阿然定制24k纯钛合金超级无敌终终终终极版桃花源专属作物菜单插件'
+description: '桃花源定制24k纯钛合金超级无敌终终终终极版桃花源专属作物菜单插件'
 author: Ranica
 version: Beta 2.1
 api-version: "1.20"
